build.gradle

```diff
@@ -10,8 +10,13 @@ repositories {
 }
 
 dependencies {
+    annotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.24'
+    compileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.24'
```

Comment:
Para quê o lombok?

Reply:
Só por visibilidade mesmo. Adicionei um log para os casos em que as colunas possuem um tipo diferente do esperado, o que resultaria em uma exceção ao tentar fazer o parse delas. Aí dou catch e logo pra gente poder debugar com mais facilidade se/quando acontecer. Como esse JAR é acoplado na aplicação do Debezium, usei lombok pra deixar o logger flexível ao que o Debezium usar.

Reply:
We decided to go with Lombok since it's a less verbose/more simple way of achieving this generic logger effect. Since the imported dependencies are only used in compile-time and for annotation processing, adding Lombok shouldn't be a problem for other projects that decide to use this.